Mathlib/Data/Set/Defs.lean

```diff
@@ -239,7 +239,7 @@ instance : SDiff (Set α) := ⟨Set.diff⟩
 /-- `𝒫 s` is the set of all subsets of `s`. -/
 def powerset (s : Set α) : Set (Set α) := {t | t ⊆ s}
 
-@[inherit_doc] prefix:100 "𝒫" => powerset
+@[inherit_doc] prefix:100 "𝒫 " => powerset
```

Comment:
Mathlib is split upon the correct use of this notation: I see two occurrences with a space and four without. That said, I don't mind this change, but perhaps it could/should be a separate PR?